Content/GUI/PassiveTree/PassiveTreeInnerPanel.cs

```diff
@@ -15,6 +15,7 @@ internal class PassiveTreeInnerPanel : SmartUIElement
 	private TreePlayer TreeSystem => Main.LocalPlayer.GetModPlayer<TreePlayer>();
 	private TreeState UiTreeState => UILoader.GetUIState<TreeState>();
 	public bool Visible = true;
```

Comment:
I'm just wondering about all the Visibles; it seems like you implimented a system to avoid that, but you didnt remove any of them, or am i missing something?

Reply:
Visible is still used to make the first Draw, Visible is true for the first tab that will be drawn. That said we could probably remove it but it would require some larger changes

Reply:
Dont you remove the element [here](https://github.com/Path-of-Terraria/PathOfTerraria/pull/148/files/a8ca609aa6d1a3aa06fc2daa598b3da46faf2ed7#diff-3a47c5274498d2062d427287482666f50f5dbf3cccfc68c28ec79350af698b1dR84)? The elements that are not children dont get drawn, no?